src/components/logout-button/logout-button.tsx

```diff
@@ -0,0 +1,61 @@
+import styled from "styled-components";
```

Comment:
re-name the file to align with component's name

Reply:
Done